app/views/figures/new.html.erb

```diff
@@ -27,6 +27,10 @@
     <%= f.text_field :claim_to_fame %>
   </div>
 
+  <div class="field" hidden="true">
+    <%= f.text_field :user_id, value: @current_user.id %>
+  </div>
```

Comment:
is this necessary? perhaps you could simply use the current_user in the controller and save the user object associated to the figure to that?